test/tests/functional/pbs_hook_timeout.py

```diff
@@ -93,7 +93,9 @@ def test_hook_send(self):
         hook_body = "import pbs\n"
         a = {'event': 'execjob_epilogue', 'enabled': 'True'}
 
-        rv = self.server.create_import_hook("test", a, hook_body)
+        rv = self.server.create_hook("test", a)
+        self.assertTrue(rv)
```

Comment:
create_hook  and import_hook functions uses manager function internally to create and import a hook and it will raise PbsManagerError if the action fails. You need not validate it explicitly

Reply:
Thanks Zulekha. I have incorporated your review comments. Please look at it.